Problem to installation

Hi! Then i start install dependency, i have this notification:

`| => yarn setup
yarn run v1.21.1
$ yarn && cd client && yarn && cd ../server && yarn
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 🔨  Building fresh packages...
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
warning " > react-scripts-ts@2.17.0" has incorrect peer dependency "typescript@2.x".
warning "react-scripts-ts > babel-jest@22.4.4" has unmet peer dependency "babel-core@^6.0.0 || ^7.0.0-0".
warning "react-scripts-ts > babel-loader@7.1.5" has unmet peer dependency "babel-core@6".
warning "react-scripts-ts > babel-preset-react-app@3.1.2" has unmet peer dependency "babel-runtime@^6.23.0".
warning "react-scripts-ts > fork-ts-checker-webpack-plugin@0.2.10" has incorrect peer dependency "typescript@^2.1.0".
warning "react-scripts-ts > ts-jest@22.0.1" has incorrect peer dependency "typescript@2.x".
[4/4] 🔨  Building fresh packages...
[-/4] ⡀ waiting...
[-/4] ⡀ waiting...
[3/4] ⡀ fsevents
warning Error running install script for optional dependency: "/Users/Master/Documents/GitHub/posenet-for-installations/client/node_modules/fsevents: Command failed.
Exit code: 1
Command: node install
Arguments: 
Directory: /Users/Master/Documents/GitHub/posenet-for-installations/client/node_modules/fsevents
Output:
node-pre-gyp info it worked if it ends with ok
node-pre-gyp info using node-pre-gyp@0.10.0
node-pre-gyp info using node@12.14.1 | darwin | x64
node-pre-gyp info check checked for \"/Users/Master/Documents/GitHub/posenet-for-installations/client/node_modules/fsevents/lib/binding/Release/node-v72-darwin-x64/fse.node\" (not found)
node-pre-gyp http GET https://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.2.4/fse-v1.2.4-node-v72-darwin-x64.tar.gz
node-pre-gyp http 404 https://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.2.4/fse-v1.2.4-node-v72-darwin-x64.tar.gz
node-pre-gyp WARN Tried to download(404): https://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.2.4/fse-v1.2.4-node-v72-darwin-x64.tar.gz 
node-pre-gyp WARN Pre-built binaries not found for fsevents@1.2.4 and node@12.14.1 (node-v72 ABI, unknown) (falling back to source compile with node-gyp) 
node-pre-gyp http 404 status code downloading tarball https://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.2.4/fse-v1.2.4-node-v72-darwin-x64.tar.gz `

And then i download models i have this error:

`downloading file at https://storage.googleapis.com/tfjs-models/weights/posenet/mobilenet_v1_050/MobilenetV1_offset_2_weights
events.js:200
      throw er; // Unhandled 'error' event
      ^

Error: incorrect header check
    at Zlib.zlibOnError [as onerror] (zlib.js:170:17)
Emitted 'error' event on Gunzip instance at:
    at errorOrDestroy (internal/streams/destroy.js:108:12)
    at Gunzip.onerror (_stream_readable.js:760:7)
    at Gunzip.emit (events.js:223:5)
    at Zlib.zlibOnError [as onerror] (zlib.js:173:8) {
  errno: -3,
  code: 'Z_DATA_ERROR'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! posenet-offline-websocket-client@0.1.0 download-models: `node downloadModels.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the posenet-offline-websocket-client@0.1.0 download-models script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/Master/.npm/_logs/2020-01-13T01_01_27_285Z-debug.log
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command`

log:

`1 verbose cli   'download-models'
1 verbose cli ]
2 info using npm@6.13.4
3 info using node@v12.14.1
4 verbose run-script [ 'predownload-models', 'download-models', 'postdownload-models' ]
5 info lifecycle posenet-offline-websocket-client@0.1.0~predownload-models: posenet-offline-websocket-client@0.1.0
6 info lifecycle posenet-offline-websocket-client@0.1.0~download-models: posenet-offline-websocket-client@0.1.0
7 warn lifecycle The node binary used for scripts is /var/folders/7r/mqs6xdgn6wq17sd5rql1rhrc0000gn/T/yarn--1578877286380-0.2159364091123699/node b>
8 verbose lifecycle posenet-offline-websocket-client@0.1.0~download-models: unsafe-perm in lifecycle true
9 verbose lifecycle posenet-offline-websocket-client@0.1.0~download-models: PATH: /usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/node-g>
10 verbose lifecycle posenet-offline-websocket-client@0.1.0~download-models: CWD: /Users/Master/Documents/GitHub/posenet-for-installations/client
11 silly lifecycle posenet-offline-websocket-client@0.1.0~download-models: Args: [ '-c', 'node downloadModels.js' ]
12 silly lifecycle posenet-offline-websocket-client@0.1.0~download-models: Returned: code: 1  signal: null
13 info lifecycle posenet-offline-websocket-client@0.1.0~download-models: Failed to exec download-models script
14 verbose stack Error: posenet-offline-websocket-client@0.1.0 download-models: `node downloadModels.js`
14 verbose stack Exit status 1
14 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
14 verbose stack     at EventEmitter.emit (events.js:223:5)
14 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
14 verbose stack     at ChildProcess.emit (events.js:223:5)
14 verbose stack     at maybeClose (internal/child_process.js:1021:16)
14 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
15 verbose pkgid posenet-offline-websocket-client@0.1.0
16 verbose cwd /Users/Master/Documents/GitHub/posenet-for-installations/client
17 verbose Darwin 18.7.0
18 verbose argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "download-models"
19 verbose node v12.14.1
20 verbose npm  v6.13.4
21 error code ELIFECYCLE
22 error errno 1`

I can start server in :3000 port, but websocket not give me information about pose. 

In localhost:8080 i have only: Required update

Previously, the server worked. I even changed the directory, but the error remains the same.

--- a/client/downloadModels.js
+++ b/client/downloadModels.js
@@ -18,9 +18,26 @@ function saveFile(downloadStream, fileName) {
   const file = fs.createWriteStream(fileName);
 
   return new Promise((resolve, reject) => {
-    const pipe = downloadStream.pipe(createGunzip()).pipe(file);
+    let pipe;
+
+    switch (downloadStream.headers['content-encoding']) {
+      case 'gzip':
+        pipe = downloadStream.pipe(zlib.createGunzip());
+        break;
+      case 'deflate':
+        pipe = downloadStream.pipe(zlib.createInflate());
+        break;
+      default:
+        pipe = downloadStream;
+        break;
+    }
 
     pipe.on('close', resolve);
+    pipe.on('error', err => {
+      reject(err)
+    });
+
+    pipe.pipe(file);
   })
 }
 

